backtrace-library/src/main/java/backtraceio/library/models/BacktraceMetricsSettings.java

```diff
@@ -0,0 +1,76 @@
+package backtraceio.library.models;
+
+import backtraceio.library.metrics.BacktraceMetrics;
+
+public class BacktraceMetricsSettings {
+    /**
+     * Backtrace metrics settings
+     * @param universeName              Backtrace universe name
+     * @param token                     Backtrace submission token
+     */
+    public BacktraceMetricsSettings(String universeName, String token) {
```

Comment:
Why do we need all these methods - we already get this information - since the user passing us the Backtrace URL that contains the universe name and token.

Why I as a user want/need to pass once again the same information?

Reply:
@konraddysput do we want to create extraction methods for these parameters? I think we can get into all kinds of string handling errors or Backtrace URL can change. User can pass the full URL to `BacktraceCredentials`, we don't have `universeName` and `token` by default.

Reply:
url has universename and token inside

Reply:
I understand URL has universename and token inside but we will need to do string regex to extract it from URL and URL pattern can change in the future. It can be quite brittle. 

Meanwhile it is trivial for user to supply universename and token once more, they will already have this information when they configure Backtrace.

Ultimately I defer to you but do we really need to make this change @konraddysput ?

Reply:
We already using the same pattern in other plugins/services so it's safe to assume the universe name pattern will stay the same. I would recommend checking how we're guessing a universe name in the backtrace-unity plugin (especially for metrics support). There are some exceptions that this url pattern already handles.

Reply:
Okay, I will use the same pattern from backtrace-unity, thank you @konraddysput !